ci-operator/config/openshift/odo/openshift-odo-master__v4.5.yaml

```diff
@@ -11,18 +25,31 @@ tag_specification:
   name: "4.5"
   namespace: ocp
 tests:
-- as: integration-e2e
-  commands: scripts/openshiftci-presubmit-all-tests.sh
-  openshift_installer_src:
+- as: integration-e2e-steps
```

Comment:
Can we remove steps from the test name ?

Reply:
As we have nested steps and we can not use same name for both the steps in line 28 and 32 https://github.com/openshift/release/pull/9431/files#diff-cf06122cc7ac596bba3930b965d2e59bR28-R32 . I have already checked that it won't work in that way. So IMO `integration-e2e-steps` best suit for this. Anyway I am open to any test name.

Reply:
The same name is being reflected in job name. 

<img width="986" alt="Screen Shot 2020-07-23 at 12 03 39 PM" src="https://user-images.githubusercontent.com/19851115/88258508-f0eb1b80-ccdd-11ea-9c21-5254c622e24f.png">

The job name we have been using since long time should not be change for no cause. So you can rename this as ```- as: integration-e2e``` and update the name https://github.com/openshift/release/pull/9431/files#diff-cf06122cc7ac596bba3930b965d2e59bR32 to ```- as: integration-e2e-steps```

Reply:
> The same name is being reflected in job name.
> 
> <img alt="Screen Shot 2020-07-23 at 12 03 39 PM" width="986" src="https://user-images.githubusercontent.com/19851115/88258508-f0eb1b80-ccdd-11ea-9c21-5254c622e24f.png">
> 
> The job name we have been using since long time should not be change for no cause. So you can rename this as `- as: integration-e2e` and update the name https://github.com/openshift/release/pull/9431/files#diff-cf06122cc7ac596bba3930b965d2e59bR32 to `- as: integration-e2e-steps` to avoid any conflict.

This makes sense to me as well 👍